CMakeLists.txt

```diff
@@ -588,13 +589,15 @@ function(fw_add_variant variant_name)
       list(APPEND ALL_VARIANT_HEXES ${FW_LANG_FINAL})
     endforeach()
     add_custom_target("${variant_name}-All-Languages" DEPENDS ${ALL_VARIANT_HEXES})
-    add_custom_command(TARGET "${variant_name}-All-Languages"
-      POST_BUILD
-      WORKING_DIRECTORY ${CMAKE_BINARY_DIR}
-      COMMAND ${CMAKE_COMMAND} -E make_directory "release"
-      COMMAND ${CMAKE_COMMAND} -E tar "cfv" "release/${variant_name}_${FN_VERSION_SUFFIX}.zip" --format=zip ${variant_name}_${FN_VERSION_SUFFIX}_en*.hex
-      BYPRODUCTS "${CMAKE_BINARY_DIR}/release/${variant_name}_${FN_VERSION_SUFFIX}.zip"
-      )
+    if(CMAKE_HOST_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
strongly suggest `IF(UNIX)` over fragile string compares

Reply:
Not working correctly